includes/admin.form.inc

```diff
@@ -263,21 +263,49 @@ function islandora_openseadragon_admin($form, &$form_state) {
  * Validates that the supplied djatoka base URL is a working djatoka install.
  */
 function djatoka_base_url_validate($element, &$form_state, $form) {
-  $url = $element['#value'];
+  $jp2_image = drupal_get_path('module', 'islandora_openseadragon') . '/test/test.jp2';
+  $image_url = file_create_url($jp2_image);
+  $server_url = $element['#value'];
+  $dimensions = '0,0,100,100';
+  $request = array(
+    '?url_ver' => 'Z39.88-2004',
```

Comment:
Question mark shouldn't be here... Would be [masking stuff](https://github.com/Islandora/islandora_openseadragon/pull/68/files#r78074028).